src/main/kotlin/apifi/codegen/RequestBodyBuilder.kt

```diff
@@ -5,7 +5,7 @@ import com.squareup.kotlinpoet.ClassName
 import com.squareup.kotlinpoet.ParameterSpec
 
 object RequestBodyBuilder {
-  fun build(bodyType: String, modelMapping: List<Pair<String, String>>): ParameterSpec =
+  fun build(bodyType: String, modelMapping: Map<String, String>): ParameterSpec =
```

Comment:
@bhimsenp could the same response type be also used for ```ResponseBodyBuilder```? We might have to brainstorm how that could be achieved, but that would make it easy

Reply:
Refactored in other commit